service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/ProxyOptions.java

```diff
@@ -0,0 +1,67 @@
+package com.microsoft.azure.sdk.iot.service;
+
+import lombok.Getter;
+
+import java.net.InetSocketAddress;
+import java.net.Proxy;
+
+/**
+ * The settings supported by this SDK when communicating to IoT Hub through a proxy. Only HTTP Proxies are supported.
+ * Authentication with the HTTP proxy is not supported.
```

Comment:
I am not sure what this means:
Only HTTP Proxies are supported. Authentication with the HTTP proxy is not supported.

Reply:
It means you can use HTTP proxies, but only if that HTTP proxy doesn't require authentication. I can reword this